lib/nerves_ssh.ex

```diff
@@ -26,18 +26,24 @@ defmodule NervesSSH do
     defstruct opts: [], sshd: nil, sshd_ref: nil
   end
 
+  defp via_tuple(name), do: {:via, Registry, {NervesSSH.Registry, name}}
```

Comment:
```suggestion
  defp via_name(name), do: {:via, Registry, {NervesSSH.Registry, name}}
```
This is typed as a `GenServer.name()`, so I typically call these `via_name/1` functions.

Reply:
Sounds reasonable (I got the name `via_tuple` from Saša Jurić's Elixir in Action)